engine/src/main/java/org/camunda/bpm/engine/impl/cfg/ProcessEngineConfigurationImpl.java

```diff
@@ -865,7 +865,8 @@
 
 
   // telemetry ///////////////////////////////////////////////////////
-  protected boolean telemetryEnabled = false;
+  /** if set to true the telemetry will be enabled from the first engine start*/
+  protected boolean telemetryInitialized = false;
```

Comment:
I would prefer an active verb for this property, e.g. `initializeTelemetry`. The reason is that this property refers to something the engine does once on startup. Other properties that use passive voice (e.g. `authorizationEnabled`) refer to a state of the engine during its lifetime.